pelias.json

```diff
@@ -39,5 +39,10 @@
         "85977539"
       ]
     }
+  },
+  "logger": {
+    "level": "http",
```

Comment:
After discussion with @TylerMatteo, decided to specify [logging level](https://github.com/winstonjs/winston#logging-levels) to be `http` to override the default level `debug`. This reduces the amount of unhelpful logs that take up disk space.